pages/insights/future-work-starts-today.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/insight
+title: The future of work starts today
+date: 2023-02-14
+author: Chuck Hardy
+position: Chief Architect
+externallink: https://www.gsa.gov/blog/2023/02/13/the-future-of-work-starts-now
+image: 
+  url: /assets/img/insights-future-work.jpg
+  alt: "Five individuals standing in the hallway of workplace innovation lab and doing ribbon cutting ceremonies. "
```

Comment:
I would defer to @brentryanjohnson on this, but maybe something like "GSA leadership opening the Workplace Innovation Lab during a ribbon cutting ceremony." might be add a little more interesting info.